Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * Inline Assembly: Improve error messages around invalid function argument count.
  * Code Generator: Use codecopy for string constants more aggressively.
  * Code Generator: Use binary search for dispatch function if more efficient. The size/speed tradeoff can be tuned using ``--optimize-runs``.
+ * Commandline interface: Report better error message for json input when 'settings' is not an object
```

Comment:
I would say preventing an internal compiler error from happening should be a bugfix and not a feature.